pallets/scoring/Readme.md

```diff
@@ -0,0 +1 @@
+#test
```

Comment:
recommended file name for Readme is `README.md` (good to stick to that as a practise).